### PR TITLE
Add Commands class with switch method and Command enum

### PR DESCRIPTION
**Pull request by cchriss123**
Redone branch with suggested changes from @safstromo @MikaelEdwartz.

### CODE REVIEW

**Review by safstromo — CHANGES_REQUESTED** [review]
Much better!
Almost done!
Id like you to rename the methods and remove "handle" :)

**Review by wassimtajeddin — APPROVED** [review]

File: src/main/java/org/fungover/haze/Commands.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+		switch (cmd) {
+			case SET -> SetCommand();
+			case GET -> GetCommand();
+			case DEL -> DelCommand();
+			case PING -> PingCommand();
+			default -> System.out.println("Invalid Command");
+		}
+	}
+
+	private void handleSetCommand() {
```
Comment (safstromo) [inline src/main/java/org/fungover/haze/Commands.java:14]: Missed some <3

**Review by safstromo — CHANGES_REQUESTED** [review]

File: src/main/java/org/fungover/haze/Commands.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+	private void handleGetCommand() {
+	}
+
+	private void handleDelCommand() {
+	}
+
+	private void handlePingCommand() {
+	}
+}
+
```
Comment (safstromo) [inline src/main/java/org/fungover/haze/Commands.java:26]: Missed some <3

**Review by safstromo — CHANGES_REQUESTED** [review]

File: src/main/java/org/fungover/haze/Commands.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
+package org.fungover.haze;
+
+public class Commands {
+	public void command(Command cmd) {
+		switch (cmd) {
+			case SET -> SetCommand();
```
Comment (jonathanjonathan001) [inline src/main/java/org/fungover/haze/Commands.java:6]: should be setCommand() right?

**Review by jonathanjonathan001 — CHANGES_REQUESTED** [review]
and also the other method calls should start with lower-case-letters. :)

File: src/main/java/org/fungover/haze/Commands.java (hunk 3 above)
Comment (cchriss123) [reply src/main/java/org/fungover/haze/Commands.java:6]: Yes I did

File: src/main/java/org/fungover/haze/Commands.java (hunk 2 above)
Comment (cchriss123) [reply src/main/java/org/fungover/haze/Commands.java:26]: Yes I did.

File: src/main/java/org/fungover/haze/Commands.java (hunk 3 above)
Comment (cchriss123) [reply src/main/java/org/fungover/haze/Commands.java]: Yes i agree. It is fixed now.

**Review by safstromo — APPROVED** [review]

**Review by wassimtajeddin — DISMISSED** [review]

**Review by jonathanjonathan001 — APPROVED** [review]
Looking great!

**Pull request closed without merging**